lib/internal/errors.js

```diff
@@ -1280,6 +1280,62 @@ E('ERR_PACKAGE_PATH_NOT_EXPORTED', (pkgPath, subpath, base = undefined) => {
       pkgPath} imported from ${base}`;
   }
 }, Error);
+E('ERR_QUICCLIENTSESSION_FAILED',
```

Comment:
I guess there is no harm in having the errors be defined all the time, even when quic is not enabled, but they become part of the API, then, harder to change, if these could be conditional on QUIC support it would give you more freedom for change and experiment.

Reply:
I've been thinking about how we can modularize the errors list and make it more manageable now that it's growing. Will give it some thought.